pkg/scheduler/eventhandlers.go

```diff
@@ -398,7 +398,7 @@ func AddAllEventHandlers(
 	)
 
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSINodeInfo) {
-		informerFactory.Storage().V1beta1().CSINodes().Informer().AddEventHandler(
+		informerFactory.Storage().V1().CSINodes().Informer().AddEventHandler(
 			cache.ResourceEventHandlerFuncs{
 				AddFunc:    sched.onCSINodeAdd,
 				UpdateFunc: sched.onCSINodeUpdate,
```

Comment:
neither onCSINodeAdd nor onCSINodeUpdate use the passed in objects, so no other changes were required